### DISCUSSION TITLE
Project templates

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#123] [Show and tell]
Hi guys. Released a new version of Revit templates, this project uses most of the code from there. Can anyone test, give feedback?) https://github.com/Nice3point/RevitTemplates/discussions/19

**Comment by jeremytammik** [comment]
i am publishing your feedback request on the blog at 11 o'clock my time, two hours from now, so people have something to do over the holidays :-)

**Comment by Nice3point** [comment]
what a surprise, thanks Jeremy)

**Comment by jeremytammik** [comment]
after all that you have done for RevitLookup, that is the least i can do in return... thank YOU! happy holidays!

**Comment by Nice3point** [comment]
merry christmas and happy new year 🎅🎄🎉
